Gemfile

```diff
@@ -56,6 +56,8 @@ gem 'yabeda-rails'
 gem 'yabeda-prometheus', '~> 0.6.1'
 gem 'yabeda-puma-plugin'
 
+gem 'webrick'
+
```

Comment:
we need this?

Reply:
Yes, this gem was excluded from Ruby since 3.0

https://www.ruby-lang.org/en/news/2020/12/25/ruby-3-0-0-released/

> The following libraries are no longer bundled gems or standard libraries. Install the corresponding gems to use these features.
>  - sdbm
>  - webrick
>  - net-telnet
>  - xmlrpc

Reply:
I wonder if we can use puma for dev and prod as well. I assume we run puma in production. Or not?

Reply:
It is a dependency for `yabeda-prometheus`, I left a comment.
It's possible to use a different web server, but we can keep it as it was for now.